src/ccnl-pkt/include/ccnl-pkt-ndntlv.h

```diff
@@ -108,6 +111,16 @@ Values          Designation
 #define NDN_Marker_Timestamp			0xFC
 #define NDN_Marker_SequenceNumber		0xFE
 
+/**
+ * @brief NDN Interest options
+ */
+struct ccnl_ndntlv_interest_opts_s {
+    int32_t nonce;              /**< Nonce value */
+    /* Selectors */
+    bool mustbefresh;           /**< MustBeFresh Selector */
+    /* Guiders */
+    uint32_t interestltime;     /**< Interest Lifetime Guider */
```

Comment:
That really feels like unnecessary. Can't we just spend three more letters and make it interestlifetime? :grimacing: